runtime/runtime-params-estimator/src/action_costs.rs

```diff
@@ -325,6 +359,10 @@ pub(crate) fn deploy_contract_byte_send_not_sir(ctx: &mut EstimatorContext) -> G
     ActionEstimation::new(ctx)
         .add_action(deploy_action(ActionSize::Max))
         .inner_iters(1) // circumvent TX size limit
+        // parameter today: 6_812_999
+        // typical estimation: < 100_000 (<100ps)
+        // setting limit to 0.1ns, we expect this to stay near 0
+        .min_gas(100_000)
```

Comment:
Would it make sense to add constants for numbers? Like

```
const GAS_100_PICOSECONDS: Gas = 100_000;
const GAS_1_NANOSECOND: Gas = 1_000_000;
```

then the code self-documents what the limit is, won’t need to worry about keeping the comment above up-to date if somebody changes the `min_gas` value.

Reply:
Yep, makes sense, updated accordingly.